observability/micrometer/src/test/resources/org/wildfly/extension/micrometer/micrometer-community-1.1.xml

```diff
@@ -0,0 +1,8 @@
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<subsystem xmlns="urn:wildfly:micrometer:community:1.1">
+    <otlp-registry endpoint="http://localhost:4318/v1/metrics"/>
```

Comment:
Please add the prometheus element to this.

Please also include all legal attributes and use an expression for each.

Basically these files should cover the entire legal xml in order to exercise the subsystem.